scripts/devnet.sh

```diff
@@ -3,5 +3,5 @@ export STARKNET_DEVNET_CAIRO_VM=rust
 if [[ $2 == "load" ]]; then  
   exec poetry run starknet-devnet --seed 0 --fork-network alpha-$1 --load-path starknet_state.dmp
 else
-  exec poetry run starknet-devnet --seed 0 --fork-network alpha-$1 --dump-on exit --dump-path starknet_state.dmp
```

Comment:
Why do we need to dump on exit? We should be interested in the dump after setup is done before any tests.